src/Command/IndexCommand.php

```diff
@@ -42,26 +43,30 @@ protected function configure(): void
             ->addOption(
                 name: 'output',
                 mode: InputOption::VALUE_REQUIRED,
-                description: 'The path to the file where the index will be printed. Leave empty for screen printing'
+                description: 'The path to the file where the index will be printed. Defaults to stdout.',
             )
             ->addOption(
                 name: 'template',
                 mode: InputOption::VALUE_REQUIRED,
-                description: 'The path to the template file to use to generate the index',
+                description: 'The path to the template file to use to generate the index.',
                 default: $this->defaultTemplate
+            )
+            ->addArgument(
+                name: 'directories',
+                mode: InputArgument::IS_ARRAY,
```

Comment:
```suggestion
                mode: InputArgument::OPTIONAL | InputArgument::IS_ARRAY,
```

Reply:
Cf. https://github.com/symfony/console/blob/6.2/Input/InputArgument.php#L78
IS_ARRAY is considered as optional